### ISSUE TITLE
Edit last submitted message rather then sending a new one?

### ISSUE THREAD

**Issue by Schizo** [numberly/matterhook#7]
I wan't to create couple of channels to track progress of processes. Therefore I would want the bot to instead of sending a new chat message, to edit the last send to the channel. is that possible?

Thanks

**Comment by ultrabug** [comment]
Hello @Schizo 

AFAIK it is not possible to edit a post from a Webhook unfortunately

**Comment by Schizo** [comment]
Thanks for the quick answer. What about changing a channels name? Is that possible?

**Comment by ultrabug** [comment]
@Schizo yep you can select the channel using the `channel` keyword argument of the `send` method

There's an example in the doc
